omnisend-for-contact-form-7/class-wpcf7-omnisend-bootstrap.php

```diff
@@ -4,7 +4,7 @@
  *
  * Plugin Name: Omnisend for Contact Form 7
  * Description: A Contact Form 7 add-on to sync contacts with Omnisend. In collaboration with Omnisnnd for WooCommerce plugin it enables better customer tracking
- * Version: 1.0.2
+ * Version: 1.1.0
```

Comment:
I think it is a major/breaking change and we should have 2.0.0 version

Reply:
I think it is is fine, same we did with Gravity forms.